CHANGELOG.md

```diff
@@ -259,6 +299,10 @@ CHANGES:
 * auth/azure: Update plugin to v0.18.1
 * secrets/openldap: update plugin to v0.13.2
 
+FEATURES:
```

Comment:
am assuming this is on purpose and we released this change last  release for 1.17.8 ?

Reply:
Nope, I put it in the wrong place